ext/birthday_commands.py

```diff
@@ -51,46 +56,49 @@ async def update(self, ctx, user_info, *date_input) -> None:
         """
         # Exit if it's not from the adequate server
         if ctx.guild.id != get_var('SERVER_ID'):
-            await ctx.send(
-                "You are not supposed to use this command on "
-                f"this server : {str(ctx.guild)}"
```

Comment:
Le str(ctx.guild) est potentiellement nécessaire, je ferais le test dans tous les cas. De mémoire il y a des cas où l'appel à "str" change l'output de l'appel à l'objet ; genre potentiellement c'est un id + jsp quoi et en indiquant str ça vient chercher le nom human readable

Reply:
Normalement dans les f-string on utilise `__format__`, pas `repr` et je lis:

> Changed in version 3.7: object.__format__(x, '') is now equivalent to str(x) rather than format(str(x), '').

donc ça devrait être équivalent : mais ça vaut le coup de vérifier !